CovertInfiltration/Src/CovertInfiltration/Classes/UISS_CovertActionRisks.uc

```diff
@@ -0,0 +1,36 @@
+class UISS_CovertActionRisks extends UIPanel;
+
+var UIText Text;
+
+simulated function InitRisks()
+{
+	InitPanel('UISS_CovertActionRisks');
+	AnchorTopRight();
+	SetPosition(-400, 0);
+	
+	Text = Spawn(class'UIText', self);
+	Text.bAnimateOnInit = false;
+	Text.InitText('Text');
+	Text.SetWidth(-X);
+}
+
+simulated function UpdateData(XComGameState_CovertAction CurrentAction)
+{
+	local array<string> Labels, Values; 
+	local string strRisks;
+	local int idx; 
+
+	CurrentAction.GetRisksStrings(Labels, Values);
+
+	for (idx = 0; idx < Labels.Length; idx++)
+	{
+		strRisks $= "<p>" $ class'UIUtilities_Text'.static.AddFontInfo(Values[idx] $ " - " $ Labels[idx], Screen.bIsIn3D) $ "</p>";
```

Comment:
Before this FOR loop, we should have a header, like "Possible Risks:" before the list of risks. This would help organize the information more clearly when we start to add other information readouts in that area of the screen.

Reply:
Although this could always just be changed when the other information goes in.

Reply:
Same here - the ui is just for testing currently